src/app/views/provisioning/confirm.html

```diff
@@ -34,9 +34,9 @@ <h1 translate>mno_enterprise.templates.dashboard.provisioning.confirm.title</h1>
     <div ng-show="(vm.singleBilling && vm.billedLocally) || vm.subscription.product.product_type != 'application'" class="table-cell">
       <div class="row th-row hidden-xs">
         <div class="cell col-sm-3">{{ 'mno_enterprise.templates.dashboard.provisioning.confirm.name' | translate }}</div>
-        <div class="cell col-sm-2">{{ 'mno_enterprise.templates.dashboard.provisioning.confirm.description' | translate }}</div>
-        <div class="cell col-sm-2">{{ 'mno_enterprise.templates.dashboard.provisioning.confirm.licenses' | translate }}</div>
-        <div class="cell col-sm-3">{{ 'mno_enterprise.templates.dashboard.provisioning.confirm.price' | translate }}</div>
+        <div class="cell col-sm-2" ng-show="vm.subscription.product_pricing.license_based">{{ 'mno_enterprise.templates.dashboard.provisioning.confirm.licenses' | translate }}</div>
```

Comment:
Do we need `.product_pricing?...` here in the event that there is no product_pricing attached to the subscription?

Reply:
Shouldn't we have 12 columns in this row instead of 10?
Also we should have conditional class based on {{vm.subscription.product_pricing.license_based}} so the total is always 12.
Same problem for the table body.